src/core/FaceControls.tsx

```diff
@@ -284,7 +285,11 @@ export const FaceControls = forwardRef<FaceControlsApi, FaceControlsProps>(
     const faceBlendshapes = faces?.faceBlendshapes?.[0]
     return (
       <FaceControlsContext.Provider value={api}>
-        {webcam && <Webcam ref={webcamApiRef} autostart={autostart} videoTextureSrc={webcamVideoTextureSrc} />}
+        {webcam && (
+          <Suspense fallback={null}>
```

Comment:
same here: is Suspense really required? I'm not sure

Reply:
Your call whether suspense should go here or in user-land around the component.